node/src/neighborhood/gossip_acceptor.rs

```diff
@@ -1655,18 +1667,14 @@ mod tests {
     }
 
     #[test]
-    fn two_debut_in_same_time_with_empty_db_handled_by_try_accept_debut() {
+    fn two_parallel_progressing_debuts_handled_by_try_accept_debut_without_introduction() {
```

Comment:
Please, this reads slightly better 

two_parallel_debuts_in_progress_handled_by_try_accept_debut_without_introduction